library/datasort.h

```diff
@@ -81,8 +81,10 @@ struct datasort_cfg {
 	struct list_head		sorted_chunks;
 	/* Result of mergesort */
 	struct datasort_chunk		*result;
-	/* Datasort directory */
-	char				*dir;
+	/* Datasort directory - chunks are merged into single sorted blob here */
+	char				*datasort_dir;
+	/* Chunks directory - if not defined, then chunks are stored in datasort_dir */
+	char				*chunks_dir;
```

Comment:
Look good to me, thank you.
Please also rename datasort_dir to something else, it is not datasort at all, it is merge or just plain dir, but not datasort.

Also, you should write a detailed description in comments to `eblob_config`
It is not enough to say that chunks will be stored there, you should explain in more details the logic for those who hardly understands the whole sorting logic. For example, it is better to explain how sorting works, what are chunks, what is merge, when sorting is used, when defrag is used and how it works.